### PR TITLE
Fix Unit Tests

### PR DESCRIPTION
**Pull request by David-Moreira**

### CODE REVIEW

**Comment by David-Moreira** [comment]
Closes #3914

Let's try this. Latest BUnit && `WaitForAssertion` BUnit API as that seems to help with any async processing that might not have been over yet.

File: Tests/Blazorise.Tests/Extensions/TestExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+using Bunit;
+using Microsoft.AspNetCore.Components;
+
+namespace Blazorise.Tests.Extensions
+{
+    internal static class TestExtensions
+    {
+        private static TimeSpan waitTime = TimeSpan.FromSeconds( 3 );
+
+        public static void WaitForElementAndClick<T>( this IRenderedComponent<T> comp, string selector ) where T : IComponent
```
Comment (stsrki) [inline Tests/Blazorise.Tests/Extensions/TestExtensions.cs:16]: Can we just have a method name `Click` that accepts the selector?

Comment (David-Moreira) [reply Tests/Blazorise.Tests/Extensions/TestExtensions.cs:16]: You mean, rename the method?

Comment (stsrki) [reply Tests/Blazorise.Tests/Extensions/TestExtensions.cs:16]: Yes.

**Review by stsrki — APPROVED** [review]

**Pull request merged**